src/containers/auth/AuthLayoutContainer.tsx

```diff
@@ -39,7 +40,7 @@ class AuthLayoutContainer extends Component<AuthLayoutContainerProps> {
       <ThemeProvider theme={stores.ui.theme}>
         <AuthLayout
           error={globalError.response}
-          pathname={stores.router.location.pathname}
+          // pathname={stores.router.location.pathname} // TODO - [TS DEBT] path name props is not used in AuthLayout
```

Comment:
If you have verified that this is not being used, please delete the property.

Reply:
I have removed it , will push it